modules/ROOT/pages/as2/as2-connector.adoc

```diff
@@ -0,0 +1,133 @@
+= AS2 Connector
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Support Category: Premium
+
+The Mule Runtime supports the AS2 protocol through the AS2 Connector. The AS2 connector enables trading partners transmit and receive AS2 messages with EDI-X12, EDIFACT, XML or binary payloads.
+
+This connector leverages the AS2 protocol, allowing Mule applications to:
+
+* Send messages, signed or unsigned, over HTTP or HTTPS, following the AS2 protocol.
+* Receive messages, also over HTTP or HTTPS, following the protocol.
+
+This document assumes that you are familiar with AS2, Mule, Anypoint Connectors, and Anypoint Studio.
+
+For hardware and software requirements, see the Connector Release Notes.
+
+
+== How to Install
+
+. In Anypoint Studio, click the Exchange icon in the Studio task bar.
+. Click Login in Anypoint Exchange and supply your Anypoint Platform username and password.
+. Search for the connector and click Install.
+. Follow the prompts to install the connector.
+
+NOTE :This connector requires http://www.oracle.com/technetwork/java/javase/downloads/jce8-download-2133166.html[Java Cryptography Extension (JCE)] installed on your Java runtime.
```

Comment:
This might be more like a pre-req unless you can't install it until after you install the connector.  ?

Reply:
I think so. I will have to check with Amit 'cause I'm pretty sure he added it.